data/json/items/gun/shot.json

```diff
@@ -57,30 +35,43 @@
     "id": "bigun",
     "copy-from": "shotgun_base",
     "type": "GUN",
-    "name": "manual autoshotgun",
-    "//": "Hard to value this thing, but considering what a $2000 pricepoint puts it on par with, that is too much.",
-    "description": "A six-barrel hand-cranked automatic shotgun made from bicycle parts.  Though a bit unwieldy, it is exceedingly powerful for such a simple machine.",
+    "name": "12-gauge gatling gun",
+    "description": "An electrically driven six barrel gatling shotgun, fed from handmade cloth belts.  Even properly mounted, this seems like an unwieldy beast, and the six separate barrels make for difficult zeroing.  The externally driven action means this is much less likely to jam.",
```

Comment:
Wouldn't this need to use UPS charges as well, if it's electrically driven?

Maybe also wouldn't necessarily be mounted then, if it's not hand-cranked; one could likely design such a device so as to be hip-fired.

Reply:
For all the reasons we didn't go very far with hip-fired M134's, I'm not doing this.

Reply:
Well, still needs to consume UPS charges either way then. So you have to have a charged battery on your vehicle if you want to fire it, just like M134 works currently.

Reply:
Sounds good!